### PR TITLE
[PrettyPrinter] Decrease digits for holdings and ref_market_value

### PR DESCRIPTION
**Pull request by piezewieter**
For readability in telegram output decrease the max digits for holdings to 4 and for the reference market value to 2

### CODE REVIEW

File: octobot_commons/pretty_printer.py [hunk 1]
```diff
@@ -149,11 +149,13 @@ def _get_markdown_pretty_portfolio_row(
...
     """
     str_currency = "{:<4}".format(currency)
-    str_holdings = "{:<12}".format(get_min_string_from_number(holdings, max_digits=6))
+    str_holdings = "{:<12}".format(get_min_string_from_number(holdings, max_digits=4))
     str_ref_market_value = "{:<12}".format("")
     if ref_market:
         str_ref_market_value = "{:<12}".format(
-            get_min_string_from_number(ref_market_value) if ref_market_value else ""
+            get_min_string_from_number(ref_market_value, max_digits=2)
++            if ref_market_value
```
Comment (Herklos) [inline octobot_commons/pretty_printer.py:157]: I don't understand why you added a `+` here

Comment (piezewieter) [reply octobot_commons/pretty_printer.py:157]: Omg, stupid typo, copy/paste error.. 🙈

File: octobot_commons/pretty_printer.py [hunk 2]
```diff
@@ -149,11 +149,13 @@ def _get_markdown_pretty_portfolio_row(
     :return: the portfolio row adapted for a markdown format
     """
     str_currency = "{:<4}".format(currency)
-    str_holdings = "{:<12}".format(get_min_string_from_number(holdings, max_digits=6))
+    str_holdings = "{:<12}".format(get_min_string_from_number(holdings, max_digits=4))
```
[the remark this answers is not in the document]
Comment (piezewieter) [reply octobot_commons/pretty_printer.py:152]: Sure, sounds better. I'll add it..

File: octobot_commons/pretty_printer.py [hunk 3]
```diff
@@ -149,12 +149,21 @@ def _get_markdown_pretty_portfolio_row(
...
-            get_min_string_from_number(ref_market_value) if ref_market_value else ""
-        )
+        if ref_market_value >= 10:
+            str_ref_market_value = "{:<12}".format(get_min_string_from_number(ref_market_value, max_digits=2))
+        elif ref_market_value >= 1:
+            str_ref_market_value = "{:<12}".format(get_min_string_from_number(ref_market_value, max_digits=4))
+        elif ref_market_value < 1:
+            str_ref_market_value = "{:<12}".format(get_min_string_from_number(ref_market_value, max_digits=6))
+        else: ""
     return f"{str_currency} {str_holdings} {str_ref_market_value}"
```
Comment (piezewieter) [inline octobot_commons/pretty_printer.py]: Fixed the second if statement, now it should work. It was inside the format() method.

**Review by piezewieter — COMMENTED** [review]
Reformatted according to black lint

File: octobot_commons/pretty_printer.py [hunk 4]
```diff
@@ -149,12 +149,34 @@ def _get_markdown_pretty_portfolio_row(
...
+        elif ref_market_value >= 1:
+            str_ref_market_value = "{:<12}".format(
+                get_min_string_from_number(ref_market_value, max_digits=4)
+            )
+        elif ref_market_value < 1:
+            str_ref_market_value = "{:<12}".format(
+                get_min_string_from_number(ref_market_value, max_digits=6)
+            )
+        else:
+            """"""
```
Comment (Herklos) [inline octobot_commons/pretty_printer.py:179]: Should we keep this else statement?

**Review by piezewieter — COMMENTED** [review]
Removed the useless else: """""" statement

**Review by Herklos — APPROVED** [review]

File: octobot_commons/pretty_printer.py [hunk 5]
```diff
@@ -142,18 +142,31 @@ def _get_row_pretty_portfolio_row(holdings, currency, ref_market, ref_market_val
     return f"{str_holdings} {currency}"
 
 
+def _get_max_digits(number):
```
Comment (Herklos) [inline octobot_commons/pretty_printer.py:145]: 👍

File: tests/test_pretty_printer.py [hunk 6]
```diff
@@ -38,6 +38,17 @@ def test_get_min_string_from_number():
     assert pretty_printer.get_min_string_from_number(101.04, max_digits=1) == "101"
     assert pretty_printer.get_min_string_from_number(100.00000000, max_digits=8) == "100"
     assert pretty_printer.get_min_string_from_number(-100.00000000, max_digits=8) == "-100"
+    # with computed max_digits
```
Comment (Herklos) [inline tests/test_pretty_printer.py:41]: 👍

**Review by Herklos — APPROVED** [review]
👍

**Pull request merged**